easybuild/easyconfigs/b/bam-readcount/bam-readcount-0.7.4-foss-2015b.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Huffman
+# The Francis Crick Institute
+easyblock = 'CMakeMake'
+
+name = 'bam-readcount'
+version = '0.7.4'
+
+homepage = 'https://github.com/genome/bam-readcount'
+description = """Count DNA sequence reads in BAM files"""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/genome/%(name)s/archive']
+
+patches = ['%(name)s-%(version)s-cmake_install_prefix.patch']
+
+builddependencies = [
+    ('SAMtools', '1.3'),
+    ('CMake', '3.4.1'),
+]
+
+configopts = " -DCMAKE_INSTALL_PREFIX=%(installdir)s "
```

Comment:
@verdurin seems like removing this was not part of 68db97d838656d94d12133f292533dc2670c42f0 like you intended...